test/python/WMCore_t/ResourceControl_t/ResourceControl_t.py

```diff
@@ -678,7 +680,10 @@ def testInsertSite(self):
         resControlPath = os.path.join(getTestBase(), "../../bin/wmagent-resource-control")
         env = os.environ
         env['PYTHONPATH'] = ":".join(sys.path)
-        cmdline = [resControlPath, "--add-Test"]
+        if PY3:
+            cmdline = ["python3", resControlPath, "--add-Test"]
+        else:
+            cmdline = [resControlPath, "--add-Test"]
```

Comment:
see this comment: https://github.com/dmwm/WMCore/issues/10548#issuecomment-863310637

Reply:
Nice!